### ISSUE TITLE
Some RNA monomers don't display in the canvas

### ISSUE THREAD

**Issue by jsanford8** [CSCIE-599/HELM-Editor-UI#80]
There are certain monomers (for instance 5fU) that will not display on the canvas. They do not display currently on the Swing editor, either, so there may be an issue with the monomers in the library.

**Comment by jsanford8** [comment]
A lot of this has been fixed (and understood) such that branch monomers should not be displayed (now a warning is shown in the console if you try.

That said, there are still a number of backbone RNA monomers that do not display correctly, such as ILR, bP, and dier. I believe there are others, too, but these are the easiest ones I can find.

@thankam - does the canvas display service have a way to display unknown monomers? Or is there some logic in there that switches based off of the name, and that might be why these aren't displayed?

**Comment by SarahL88** [comment]
Justin, 
Yes, I the canvas display service graphs based on three types: 1) Nucleotide, 2) Peptide, 3) CHEM.
You can see an example of this in main.js, in the makeLinearGraph() method.  That method directs sequences for processing based on these three types.  If a sequence doesn't fall into one of these categories, it's not processed.
Types are assigned in the main.js 'getType()' method.
Sarah

**Comment by jsanford8** [comment]
Found the issue here - the monomer lLR is actually an l, not an i/I at the beginning, which was causing it not to display. I also added bP and dier to the ribose check, in lieu of us using the monomer library to check these things (which we should do in the future).

I've issued a very minor PR - @thankam / @SarahL88 could one of you check it over very quickly to make sure I didn't break anything and merge it in? Then I can continue working on the connections, which shouldn't be too much longer (hopefully today)
